### DISCUSSION TITLE
Inquiry Regarding Privacy Policy in XamarinCommunityToolkit

### DISCUSSION THREAD

**Discussion by Nonaba002** [xamarin/XamarinCommunityToolkit#2000] [Q&A] [answered]
Hello,
I'm currently considering the XamarinCommunityToolkit for integration into our project. As part of our diligence regarding data privacy and compliance, I'd like to inquire about the privacy policy associated with this toolkit.
Could someone from the maintainers' team please provide information on whether XamarinCommunityToolkit collects user data or sensitive information during its usage or integration? Understanding its data collection practices is essential for us to ensure alignment with our project's privacy policies.
Your response regarding this matter will greatly influence our decision-making process in adopting this toolkit.
Thank you for your attention to this inquiry.
Best regards,
Nonaba002

**Comment by bijington** [comment, marked as answer]
I can confirm that the toolkit does not collect any data. All of the source code is here on GitHub for you to review and confirm for yourself though
